src/test/fuzz/coins_view.cpp

```diff
@@ -43,13 +44,12 @@ void initialize_coins_view()
     g_setup = testing_setup.get();
```

Comment:
Unrelated to this PR, but is there a reason this is using the full testing setup?
```diff
diff --git a/src/test/fuzz/coins_view.cpp b/src/test/fuzz/coins_view.cpp
index 8f3e357a84..c7563ae9f4 100644
--- a/src/test/fuzz/coins_view.cpp
+++ b/src/test/fuzz/coins_view.cpp
@@ -30 +30 @@ namespace {
-const TestingSetup* g_setup;
+const BasicTestingSetup* g_setup;
@@ -42 +42 @@ void initialize_coins_view()
-    static const auto testing_setup = MakeNoLogFileContext<const TestingSetup>();
+    static const auto testing_setup = MakeNoLogFileContext<>(ChainType::MAIN);
```